src/main/java/uk/yermak/audiobookconverter/MediaInfoProxy.java

```diff
@@ -61,6 +63,11 @@ public void moveDown() {
         this.getMediaInfo().moveDown();
     }
 
+    @Override
+    public List<MediaInfo> getMedia() {
+        return List.of(getMediaInfo());
```

Comment:
Could you please change to delegate please, as proxy should not bring additional logic just wrapping

Reply:
Fixed